src/openrct2/actions/GameAction.cpp

```diff
@@ -146,7 +146,7 @@ namespace GameActions
             return "cl";
         else if (network_get_mode() == NETWORK_MODE_SERVER)
             return "sv";
-        return "";
+        return "cl";
```

Comment:
Why? It's neither server nor client, so perhaps it could be marked `sp` for single player here?